src/Annotation/README.md

```diff
@@ -0,0 +1,236 @@
+# Annotation feature
+
+This feature provides a system to declare annotations in docComment and 
+retrieve it into php classes.
+
+## Declaration 
+_Rules to be observed when declaring an annotation in the docComment_
+##### Identification
```

Comment:
Codacy found an issue: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6283010)